### PR TITLE
[289509] Test parameter validation plug and its error handling

### PR DESCRIPTION
**Pull request by Double-oxygeN**
https://acsmine.tok.access-company.com/redmine/issues/289509

### CODE REVIEW

File: web/controller/error.ex [hunk 1]
```diff
@@ -59,6 +59,22 @@ defmodule Testgear.Controller.Error do
     end)
   end
 
+  def parameter_validation_error(conn, parameter_type, {reason_type, mods}) do
+    raise_if_told(conn, fn ->
+      Conn.json(conn, 400, %{
+        error: "parameter_validation_error",
```
[the remark this answers is not in the document]
Comment (Double-oxygeN) [reply web/controller/error.ex]: Fixed at 6089e58.

File: test/error_handler_test.exs [hunk 2]
```diff
@@ -135,4 +135,34 @@ defmodule Testgear.ErrorHandlerTest do
     assert res.status == 400
     assert Poison.decode!(res.body) == %{"error" => "bad_executor_pool_id"}
   end
+
+  test "execute custom error handler when parameter validation fails" do
```
[the remark this answers is not in the document]
Comment (Double-oxygeN) [reply test/error_handler_test.exs:139]: Thank you. Fixed at 1c15c17.

File: test/error_handler_test.exs [hunk 3]
```diff
@@ -135,4 +135,34 @@ defmodule Testgear.ErrorHandlerTest do
...
+    [
+      {"/params_validation/0?foo=2", ~S({"foo": 3}), %{"content-type" => "application/json", "x-foo" => "4"}, %{"foo" => "5"}, "path_matches", "invalid_value", ["Elixir.Testgear.Controller.ParamsValidation.PathMatches", ["Elixir.Croma.PosInteger", "foo"]]},
```
[the remark this answers is not in the document]
Comment (Double-oxygeN) [reply test/error_handler_test.exs]: Fixed at 1c15c17.

**Review by aYukiSekiguchi — APPROVED** [review]
LGTM

**Pull request merged**